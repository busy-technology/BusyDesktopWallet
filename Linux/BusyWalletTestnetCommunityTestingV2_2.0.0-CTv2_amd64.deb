version https://git-lfs.github.com/spec/v1
oid sha256:43ecaa1ea0698c67ce409218c53d7e8e7d03b338a0b5ce756c52d4f8fc5b6b92
size 83679092
