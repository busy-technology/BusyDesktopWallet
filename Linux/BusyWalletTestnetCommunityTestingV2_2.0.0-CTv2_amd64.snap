version https://git-lfs.github.com/spec/v1
oid sha256:083aa5ccfedcb8b787cb5b3104d6a2ecf3fca681c120586d34a7d31f6e1acf19
size 133992448
